include/dlaf_c/grid.h

```diff
@@ -0,0 +1,22 @@
+//
+// Distributed Linear Algebra with Future (DLAF)
+//
+// Copyright (c) 2018-2023, ETH Zurich
+// All rights reserved.
+//
+// Please, refer to the LICENSE file in the root directory.
+// SPDX-License-Identifier: BSD-3-Clause
+//
+#pragma once
+
+#include <mpi.h>
+
+extern "C" {
+MPI_Comm Cblacs2sys_handle(int ictxt);
+void Cblacs_get(int ictxt, int inum, int* comm);
+extern "C" void Cblacs_gridinfo(int ictxt, int* np, int* mp, int* px, int* py);
```

Comment:
What was the conclusion about these? Can they not be accessed through a scalapack header? It looks like it should at least be possible to move them to `grid.cpp` if they need to be declared?

Reply:
The official scalapack does not have header files (please correct me if I am wrong). I searched in the code and the BLACS library which is written in C does not export functions and structures. scalapack been in FORTRAN, only export modules files (when we are lucky) only. 
@RMeli  pointed out in a private message that MKL provides an interface which is good but all the other implementations do not necessarily. the Cblacs functions should be declared explicitly in that case. COSMA does the same (i took it from cosma initially).

Reply:
See comment below.